src/main/webapp/app/course/manage/course-management-tab-bar/course-management-tab-bar.component.ts

```diff
@@ -136,6 +140,9 @@ export class CourseManagementTabBarComponent implements OnInit, OnDestroy, After
         this.courseSub = this.courseManagementService.find(courseId).subscribe((courseResponse) => {
             this.course = courseResponse.body!;
             this.isCommunicationEnabled = isCommunicationEnabled(this.course);
+
+            this.setExistingSummaryEntries();
+            this.fetchAndSetCourseDeletionSummary();
```

Comment:
This is "only" the instructor perspective, so we do not need to optimize to the same extend as for the student perspective.

However, a course deletion seem to be an edge case for me, I think we should only fetch this information when we want to delete really want to delete the course and not whenever we load the course overview page.

Reply:
Fair point, reasoning was to not change the deletion dialog too much :)